editor.planx.uk/src/@planx/components/Feedback/Public/Public.tsx

```diff
@@ -45,18 +43,23 @@ const FeedbackComponent = (
   return (
     <Card handleSubmit={formik.handleSubmit}>
       <CardHeader title={props.title} />
```

Comment:
Minor bug: when I add more information/help text in the editor modal, I'm not seeing the "More information" link appear as I would expect on public component! Might want to double check how other component headers are handling this!